frontend/components/text-classifier/header/TextClassificationHeader.vue

```diff
@@ -148,19 +149,36 @@ export default {
       });
     },
     async onClear(records) {
-      const clearedRecords = records.map((record) => {
-        return {
-          ...record,
-          currentAnnotation: null,
-          annotation: null,
-          selected: true,
-          status: "Edited",
-        };
-      });
-      this.updateRecords({
-        dataset: this.dataset,
-        records: clearedRecords,
-      });
+      if (this.isMultiLabel) {
+        const clearedRecords = records.map((record) => {
+          return {
+            ...record,
+            currentAnnotation: null,
+            annotation: null,
+            selected: true,
+            status: "Edited",
+          };
+        });
+        this.updateRecords({
+          dataset: this.dataset,
+          records: clearedRecords,
+          persistBackend: true,
+        });
+      } else {
+        const clearedRecords = records.map((record) => {
+          return {
+            ...record,
+            currentAnnotation: null,
+            annotation: null,
+            selected: false,
+            status: "Default",
+          };
+        });
+        await this.resetAnnotations({
+          dataset: this.dataset,
+          records: clearedRecords,
+        });
+      }
```

Comment:
1/ duplicated code : add a function to return the clearedRecords
2/ the existing funciton `updateRecords` doesn't a `await` keyword ? I ask because there was already the `async` keyword before the onClear function